open-craft/xblock-group-project-v2
# (WIP) Performance tuning

What's in this PR:
1. Switching to using `add_javascript` and `add_css` instead of `add_javascript_url` and `add_css_url`
2. Fixing flaky tests

*Testing instructions:*
1. Smoke test - open group work page and make sure js parts work and GP is properly styled
2. Run integration tests (probably a couple of times; they work fine in xvfb so far)
